ramp4-pcar4/ramp4-pcar4
# Small File Formats

### Related Item(s)

https://github.com/ramp4-pcar4/ramp4-pcar4/issues/2388

### Changes
- Adds layers to support Zipped GeoJson, Flatgeobuf, Zipped Flatgeobuf

Things that are outstanding (will be donethanksed in other PRs)
- Wizard support for new file types (#2422)
- Mini icon integration with new types (#2335)

### QA Testing

Please test against `main` branch https://ramp4-pcar4.github.io/ramp4-pcar4/main/demos/enhanced-samples.html

### Testing

1. Open Enhanced Sample 5 (Simple File Layers).
2. Observe new test layers "Zipped GeoJson" and "Zipped FlatGeobuf".
3. Interact with layers as you normally would. Look for anything behaving badly.


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2390)
